src/generic/problem.h

```diff
@@ -3112,24 +3115,58 @@ namespace oomph
     /// Max. residual when Newton solver died
     double maxres;
 
+  public:
     /// Default constructor, does nothing
-    NewtonSolverError() : linear_solver_error(false), iterations(0), maxres(0.0)
+    NewtonSolverError()
+      : OomphLibError("There was an error in the Newton Solver.",
+                      OOMPH_CURRENT_FUNCTION,
+                      OOMPH_EXCEPTION_LOCATION),
+        linear_solver_error(false),
+        iterations(0),
```

Comment:
"iterations" uppercase too. Not sure where it's defined, actually.

Reply:
Fixed!